src/bin/pgcopydb/string_utils.h

```diff
@@ -16,13 +16,22 @@
 
 /* maximum decimal int64 length with minus and NUL */
 #define INTSTRING_MAX_DIGITS 21
+#define UINT32STRING_MAX_DIGITS 10
```

Comment:
Not used.

Reply:
Yeah I was about to push but now removing it.